vllm/v1/core/sched/scheduler.py

```diff
@@ -968,7 +968,7 @@ def shutdown(self) -> None:
             self.kv_event_publisher.shutdown()
 
     ########################################################################
-    # P/D Related Methods
+    # KV Connector Related Methods
```

Comment:
I think P/D (Prefill/Decode) is meaningful and should not be deleted entirely. Maybe we add the explanation for P/D?

Reply:
@yewentao256 the connector API that the scheduler interfaces with is more generic than just P/D, the same primitives can be used for example by connectors that implement offloading and reloading from a distributed kvcache. So here "KV connector" methods is more appropriate (P/D disagg is just one application of those).